CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+## Breaking Changes
+- liboembed is renamed to liboembedtls.
+- The `openenclave-config.cmake` and `openenclave-lvi-mitigation-config.cmake` no longer include liboembedtls as `PUBLIC` link requirement for liboeenclave.
+    - Enclave apps that build with CMake and use the Open Enclave cmake configurations must now explicitly include `openenclave::oecryptombedtls` when linking `openenclave::oeenclave`.
```

Comment:
The name `oecryptombedtls` indicates that tls functionality is being added to the enclave, which not all enclaves might want to do.
Did you mean `oecryptombed` ?